orange3/si.yaml

```diff
@@ -13046,89 +13046,89 @@ widgets/visualize/owmosaic.py:
 widgets/visualize/ownomogram.py:
     class `SortBy`:
         def `items`:
-            No sorting: null
-            Name: null
-            Absolute importance: null
-            Positive influence: null
-            Negative influence: null
+            No sorting: "Brez razvrščanja"
+            Name: "Ime"
+            Absolute importance: "Absolutna pomembnost"
+            Positive influence: "Pozitiven vpliv"
+            Negative influence: "Negativen vpliv"
     class `DotItem`:
-        'ul {margin-top: 1px; margin-bottom: 1px;}': null
+        'ul {margin-top: 1px; margin-bottom: 1px;}': false
         ? "<html><head><style type=\"text/css\">{}</style>\n    </head><body><b>{}</b><hr/>{}</body></html>\n\
             \    "
-        : null
+        : false
     class `ProbabilitiesDotItem`:
         def `get_tooltip_text`:
-            'Total: {} <br/>Probability: {:.0%}': null
+            'Total: {} <br/>Probability: {:.0%}': 'Skupno: {} <br/>Verjetnost: {:.0%}'
     class `DiscreteMovableDotItem`:
         def `get_tooltip_text`:
-            'Points: {}': null
-            '{}: {:.0%}<br/>': null
+            'Points: {}': 'Točke: {}'
+            '{}: {:.0%}<br/>': false
         def `_get_tooltip_labels_with_percentages`:
-            <: null
-            '&lt;': null
+            <: false
+            '&lt;': false
     class `GraphicsColorAnimator`:
         def `__init__`:
-            brushColor: null
+            brushColor: false
     class `ContinuousItemMixin`:
         def `get_tooltip_text`:
-            'Points: {}': null
-            'Value: {}': null
+            'Points: {}': 'Točke: {}'
+            'Value: {}': 'Vrednost: {}'
     class `ProbabilitiesRulerItem`:
         def `__init__`:
-            Total: null
-            ' ': null
+            Total: "Skupno"
+            ' ': false
     class `OWNomogram`:
-        Nomogram: null
-        ' Nomograms for Visualization of Naive Bayesian': null
-        ' and Logistic Regression Classifiers.': null
-        icons/Nomogram.svg: null
+        Nomogram: true
+        ' Nomograms for Visualization of Naive Bayesian': "Nomogram za prikaz modelov"
+        ' and Logistic Regression Classifiers.': "naivni Bayes ter logistične regresije."
```

Comment:
"naivni Bayes in logistična regresija" ali pa "naivnega Bayesa in logistične regresije" (sklon)